apps/captain_fact/test/captain_fact/videos/videos_test.exs

```diff
@@ -27,6 +27,14 @@ defmodule CaptainFact.VideosTest do
       assert video.title == "__TEST-TITLE__"
     end
 
+    test "fetch captions" do
+      user = insert(:user, is_publisher: true)
+      {:ok, video} = Videos.create!(user, test_url())
```

Comment:
You can use the special factory in tests and dev to insert a "fake" video (see `apps/db/test/support/factory.ex`), it will automatically create a user for you:

```elixir
video = DB.Factory.insert(:video)
```

(documentation ExMachina factories: https://github.com/thoughtbot/ex_machina)